two_muscle_case/cuboid_muscles_with_prestretch/settings_contraction_with_prestretch.py

```diff
@@ -37,6 +37,8 @@
   print("scenario_name: {}".format(scenario_name))
   print("force 1: {}".format(force_1))
   print("force 2: {}".format(force_2))
+if len(sys.argv) > 5:
```

Comment:
Why?

Reply:
Lukas clarified this is the individuality parameter defined in `setup_BayesOpt_cuboid_2D.py`.